### DISCUSSION TITLE
Is a MkDocs theme being considered?

### DISCUSSION THREAD

**Discussion by viniciusferrao** [carbon-design-system/carbon#11814] [Ideas]
Hello, is anyone considering or developing a [MkDocs](https://www.mkdocs.org) theme with Carbon Design System? Does IBM have any interest on this?

**Comment by durera** [comment]
Very basic, but here's one I made earlier: 

- https://github.com/durera/mkdocs-carbon
- https://pypi.org/project/mkdocs-carbon/

![image](https://github.com/user-attachments/assets/30dc813f-3b58-4f85-952e-ce9f8b78856c)

**Comment by viniciusferrao** [comment]
That's awesome David.

I'll take a look.
